mkosi.conf.d/10-centos-fedora/mkosi.conf

```diff
@@ -59,3 +59,4 @@ Packages=
         tpm2-tools
         veritysetup
         vim-common
+        virt-firmware
```

Comment:
Please add to the other distributions as well. python3-virt-firmware on debian/ubuntu. virt-firmware on Arch/opensuse